packages/builder/tests/index.test.ts

```diff
@@ -326,6 +343,70 @@ describe('Linter Builder', () => {
         }),
       );
     });
+
+    it('should not log warnings if the quiet flag was passed', async () => {
+      mockReports = [
+        {
+          errorCount: 0,
+          warningCount: 4,
+          results: [],
+          messages: [
+            {
+              line: 0,
+              column: 0,
+              message: 'Mock warning message',
+              severity: 2,
```

Comment:
Something seems wrong here, severity `2` is an error, not a warning

Reply:
Yeah, the issue was that `mockFormatter.format()` returned string[] (`[ 'Mock warning message' ]`) but in the test, it compared with the string literal (`'Mock warning message'`). It's fixed now.